plugins/content_distribution/providers/tvinci/TvinciDistributionPlugin.php

```diff
@@ -2,7 +2,8 @@
 /**
  * @package plugins.tvinciDistribution
  */
-class TvinciDistributionPlugin extends KalturaPlugin implements IKalturaPermissions, IKalturaEnumerator, IKalturaPending, IKalturaObjectLoader, IKalturaContentDistributionProvider, IKalturaConfigurator
+class TvinciDistributionPlugin extends KalturaParentContributedPlugin implements IKalturaPermissions, IKalturaEnumerator, IKalturaPending, IKalturaObjectLoader,
```

Comment:
But this class is abstract and cannot be part of the plugin list - if I understand it correctly the dependency can work only on plugins that can appear in the plugins.ini and this one cannot (same as KalturaApplicationPlugin)